src/client/main.tsx

```diff
@@ -24,22 +26,45 @@ export class Main extends React.Component<{}, ClientState> {
       stage: Stages.SPLASH,
       numPlayersInLobby: 0,
     };
+    let pathName = window.location.pathname.substring(1); // Cutting off preceding '/'
+    if (pathName.length === GAME_CODE_LENGTH) {
```

Comment:
`/game/a5wem` isn't too long, can even do somtehing like `/g/a5wem`.  doesn't have to be 5 characters, just not too long like a full hashcode